cluster/addons/metrics-server/aggregated-metrics-reader.yaml

```diff
@@ -0,0 +1,14 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: system:aggregated-metrics-reader
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
```

Comment:
does the metrics server allow namespaced access to pod metrics? if not, these permissions should not be added to the admin/edit/view roles, which are intended to be used by namespaced users

Reply:
Those metrics are namespaced:

```json
/apis/metrics.k8s.io/v1beta1/namespaces/kube-system/pods

{

    "kind": "PodMetricsList",
    "apiVersion": "metrics.k8s.io/v1beta1",
    "metadata": {
        "selfLink": "/apis/metrics.k8s.io/v1beta1/namespaces/kube-system/pods"
    },
    "items": [
        {
            "metadata": {
                "name": "kube-proxy-tgq75",
                "namespace": "kube-system",
                "selfLink": "/apis/metrics.k8s.io/v1beta1/namespaces/kube-system/pods/kube-proxy-tgq75",
                "creationTimestamp": "2018-12-26T22:13:45Z"
            },
            "timestamp": "2018-12-26T22:12:47Z",
            "window": "30s",
            "containers": [
                {
                    "name": "kube-proxy",
                    "usage": {
                        "cpu": "2584225n",
                        "memory": "27296Ki"
                    }
                }
            ]
        },
        {
            "metadata": {
                "name": "privileged-pod",
                "namespace": "kube-system",
                "selfLink": "/apis/metrics.k8s.io/v1beta1/namespaces/kube-system/pods/privileged-pod",
                "creationTimestamp": "2018-12-26T22:13:45Z"
            },
            "timestamp": "2018-12-26T22:12:49Z",
            "window": "30s",
            "containers": [
                {
                    "name": "busybox",
                    "usage": {
                        "cpu": "0",
                        "memory": "2236Ki"
                    }
                }
            ]
        }
]}

Reply:
Also https://github.com/kubernetes-incubator/metrics-server/blob/ae42a7df60ec08f1fe653ed99f504e85cd34d5cd/pkg/storage/podmetrics/reststorage.go#L162-L164